lib/fxpay/index.js

```diff
@@ -39,6 +34,7 @@
 
 
   exports.validateAppReceipt = function validateAppReceipt(callback) {
+    settings.initialize();
```

Comment:
the first thing that would happen without this call is probably an 'undefined has no method X' error. Sigh, javascript. I can't really think of a good safety net. I think it's something we'll just need to enforce via code review. The old code actually called ``configure()`` when the fxpay ``index.js`` file *loaded*. I think this is bad practice because it could introduce unexpected latency to things like Fireplace in the future. Also, it was odd timing because a custom adapter may not have been set up. 

I'm open to other ideas. Maybe when we switch to promises, a better pattern will emerge.